src/CMakeLists.txt

```diff
@@ -49,6 +49,9 @@ if(NOT CMAKE_BUILD_TYPE STREQUAL Debug)
       CMAKE_C_FLAGS_${build_type_upper})
     string(REGEX REPLACE "(^| )[/-]D *NDEBUG($| )" " " "${flags}" "${${flags}}")
   endforeach()
+else()
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${CMAKE_CXX_FLAGS_DEBUG} -O0 -g" )
```

Comment:
user can always customize `CMAKE_CXX_FLAGS` and `CMAKE_C_FLAGS`  explicitly.  so again, i'd suggest remove this change.